plugins/doc_fragments/rundeck.py

```diff
@@ -29,4 +29,5 @@ class ModuleDocFragment(object):
     description:
       - Rundeck User API Token.
     required: true
+    aliases: ["token"]
```

Comment:
Why introduce an alias? Aliases should be avoided.

Reply:
I added the `token` alias because the rundeck_project and rundeck_acl_policy modules are using like that.
Is it better to not add the alias and document it as a breaking_change?

Reply:
This must not be a breaking change, otherwise it won't get merged.

But in that case I would not add the alias unconditionally, but only add it for the modules that used `token` before.

Reply:
Got it!